frontend/src/pages/modelRegistry/screens/RegisteredModels/RegisteredModelListView.tsx

```diff
@@ -105,12 +105,19 @@ const RegisteredModelListView: React.FC<RegisteredModelListViewProps> = ({
     </ToolbarGroup>
   );
 
+  const clearAllFilters = resetFilters;
```

Comment:
I don't think there is a need to create a new variable when we can just use `resetFilters` everywhere. Also, I see `() => setSearch('')` present in the same file numerous times when `resetFilters` function is doing the same thing. Can we replace all of them with `resetFilters`? Otherwise, we can rename `resetFilters` to `clearAllFilters` but all `() => setSearch('')` should be replaced with that function.